
Let's remove the annoying "not checked and I mount read-only" code ...

  - Rene Rebe <rene@exactcode.de>

--- linux/fs/hfsplus/super.c.orig	2004-10-13 15:32:30.173269240 +0200
+++ linux/fs/hfsplus/super.c	2004-10-13 15:32:25.263015712 +0200
@@ -248,9 +248,9 @@
 		if ((vhdr->attributes & cpu_to_be32(HFSPLUS_VOL_INCNSTNT)) ||
 		    !(vhdr->attributes & cpu_to_be32(HFSPLUS_VOL_UNMNT))) {
 			printk("HFS+-fs warning: Filesystem was not cleanly unmounted, "
-			       "running fsck.hfsplus is recommended.  leaving read-only.\n");
-			sb->s_flags |= MS_RDONLY;
-			*flags |= MS_RDONLY;
+			       "running fsck.hfsplus is recommended.\n");
+			/*sb->s_flags |= MS_RDONLY;
+			  *flags |= MS_RDONLY;*/
 		} else if (vhdr->attributes & cpu_to_be32(HFSPLUS_VOL_SOFTLOCK)) {
 			printk("HFS+-fs: Filesystem is marked locked, leaving read-only.\n");
 			sb->s_flags |= MS_RDONLY;
@@ -335,8 +335,8 @@
 	    !(vhdr->attributes & cpu_to_be32(HFSPLUS_VOL_UNMNT))) {
 		if (!silent)
 			printk("HFS+-fs warning: Filesystem was not cleanly unmounted, "
-			       "running fsck.hfsplus is recommended.  mounting read-only.\n");
-		sb->s_flags |= MS_RDONLY;
+			       "running fsck.hfsplus is recommended.\n");
+		/*sb->s_flags |= MS_RDONLY;*/
 	} else if (vhdr->attributes & cpu_to_be32(HFSPLUS_VOL_SOFTLOCK)) {
 		if (!silent)
 			printk("HFS+-fs: Filesystem is marked locked, mounting read-only.\n");
